config/initializers/sentry.rb

```diff
@@ -1,5 +1,6 @@
 if ENV.fetch("SENTRY_DSN", nil).present?
   Sentry.init do |config|
+    config.environment = HostEnv.env_name
```

Comment:
HostEnv doesn't exist in this app. You'll need to set an env var in deployment.yaml and deployment-worker.yaml based on .Values.environment, and reference that env var from the ruby code.

Reply:
Using ENV.fetch